packages/sd-jwt-vc/src/index.ts

```diff
@@ -34,4 +42,47 @@ export class SDJwtVcInstance extends SDJwtInstance<SdJwtVcPayload> {
       }
     }
   }
+
+  /**
+   * Verifies the SD-JWT-VC.
+   */
+  async verify(
+    encodedSDJwt: string,
+    requiredClaimKeys?: string[],
+    requireKeyBindings?: boolean,
+  ) {
+    // Call the parent class's verify method
+    const result = await super
+      .verify(encodedSDJwt, requiredClaimKeys, requireKeyBindings)
+      .then((res) => {
+        return { payload: res.payload as SdJwtVcPayload, header: res.header };
+      });
+
+    if (result.payload.status) {
+      //checks if a status field is present in the payload based on https://www.ietf.org/archive/id/draft-ietf-oauth-status-list-02.html
+      if (result.payload.status.status_list) {
+        // fetch the status list from the uri
+        result.payload.status.status_list.uri;
+        if (!this.userConfig.statusListFetcher) {
+          throw new SDJWTException('Status list fetcher not found');
+        }
+        // fetch the status list from the uri
+        const statusListJWT = await this.userConfig.statusListFetcher(
+          result.payload.status.status_list.uri,
+        );
+        // get the status list from the status list JWT
+        const statusList = getListFromStatusListJWT(statusListJWT);
+        const status = statusList.getStatus(
+          result.payload.status.status_list.idx,
+        );
+        // validate the status
+        if (!this.userConfig.statusValidator) {
+          throw new SDJWTException('Status validator not found');
+        }
```

Comment:
Can we have a default status list implementation? There's some values defined in the base specification which I think will cover 99% of cases, so most users will never have to provide a custom validator i think

Reply:
That's a good idea. The default one makes it much easier to use

Reply:
I think It's okay to make one after this PR :)

Reply:
I don't see why we need to make the simple things harder because we're not covering all cases? If you do the complex flow you probably want to do something custom anyway, but if you're just getting started having defaults is really helpful.

I don't see a reason why not. 

(Maybe we need to have some document where we define the goals and non-goals of this library. A lot of the discussions we're having in this PR seems to be revolved around assumptions in what it should do and how things should work)

My assumption:
- cross platform (including react native)
- easy to get started, customizable if you need it / sane defaults (I think we may be misaligned on this topic @cre8?)
- ..?

Reply:
I added default implementations for fetcher and status validation that can be overwritten.